frontend/public/index.html

```diff
@@ -16,7 +16,7 @@
     <!-- Footer -->
     <footer class="footer">
       <div class="container">
-        <p class="text-muted">&copy; Ryan Noelk - 2016 - <a href="https://ryannoelk.com">ryannoelk.com</a></p>
+        <p class="text-muted">Created with <a href="https://github.com/RyanNoelk/OpenEats">OpenEats</a></p>
```

Comment:
But now that I realise, this lives outside the scope of the app, not sure how we'd translate this... I'll dig around a bit and see.